demisto_sdk/commands/validate/tests/BC_validators_test.py

```diff
@@ -727,3 +732,84 @@ def test_IsBreakingContextOutputBackwardsValidator_is_valid(
             for result, expected_msg in zip(results, expected_msgs)
         ]
     )
+
+
+def create_dummy_integration_with_context_path(
+    command_name: str, context_path: str
+) -> Integration:
+
+    integration = create_integration_object()
+    command = Command(name=command_name)
+    command.outputs = [Output(contextPath=context_path)]
+    integration.commands = [command]
+
+    return integration
+
+
+def test_IsContextPathChangedValidator():
+    """
+    Given
+    integration and old integration.
+        - Case 1: no changes in context path - a valid integration
+        - Case 2: context path has been changed - an invalid integration
+    When
+    - Calling the IsContextPathChangedValidator.
+    Then
+        - Make sure the validation fail when it needs to and the right error message is returned.
+        - Case 1: Shouldn't fail any.
+        - Case 2: Should fail.
+    """
+    command_name = "command"
+    old_context_path = "test.test"
+
+    new_integration = create_dummy_integration_with_context_path(
+        command_name=command_name, context_path=old_context_path
+    )
+    old_integration = create_dummy_integration_with_context_path(
+        command_name=command_name, context_path=old_context_path
+    )
+
+    new_integration.old_base_content_object = old_integration
+
+    # integration is valid so we get an empty list
+    assert not IsContextPathChangedValidator().is_valid(content_items=[new_integration])
+
+    new_integration.commands[0].outputs[0].contextPath = f"{old_context_path}1"
+
+    # integration is invalid, so we get a list which contains ValidationResult
+    errors = IsContextPathChangedValidator().is_valid(content_items=[new_integration])
+    assert errors, "Should have failed validation"
+    assert old_context_path in errors[0].message
+    assert errors[0].message.startswith(
```

Comment:
If we check the message, it's best to check the dynamic part (the paths) rather than the prefix

Reply:
we do it, but @YuvHayun asked to check the prefix too

Reply:
where do we do it?